cms/core/tests/test_models.py

```diff
@@ -20,14 +20,6 @@ def test_contactdetails_trims_trailing_whitespace_on_save(self):
 
         self.assertEqual(details.name, "Retail")
 
-    def test_contactdetails_uniqueness_validation__with_name_case_variation(self):
```

Comment:
These had to be removed due to the fact that we currently have to use `unique_together` which, afaik, cannot be combined with `Lower()`. Happy to revisit if a better solution exists.

Reply:
Addressed in [2cae984](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/2cae984bfda9e7a3ecbcef228817fe6aae35c395)